tuiview/viewerwidget.py

```diff
@@ -779,13 +779,20 @@ def mouseReleaseEvent(self, event):
 
             layer = self.layers.getTopRasterLayer()
             if layer is not None:
-                if selectionsize == 0:
-                    fraction = 0.5 # they just clicked
+                if selectionsize < 9:  
```

Comment:
Can you change `9` to a module level global? `MIN_SELECTION_SIZE` or something similar?

Reply:
done